build.yaml

```diff
@@ -1204,3 +1203,55 @@ job_type:
                    *package_jenkins_osx_yosemite_box ]
           }
       timeout: 120
+
+#-----------------------------------------------------------------------------#
+# View definitions below this point
+#-----------------------------------------------------------------------------#
+
+views:
+  trial:
+    description: 'All trial jobs'
+    regex: 'run_(?i)trial_.*.*'
+  acceptance:
+    description: 'All Acceptance jobs'
+    regex: 'run_(?i)acceptance_.*.*'
+  client:
+    description: 'All client instalation jobs'
+    regex: 'run_(?i)client_.*.*'
+  code_quality:
+    description: 'All code quality jobs'
+    regex: 'run_(?i)lint.*.*'
+  docs:
+    description: 'Documentation jobs'
+    regex: 'run_(?i)sphinx.*.*'
+  cinder:
+    description: 'cinder jobs'
+    regex: '.*_(?i)cinder_.*.*'
+  ebs:
+    description: 'ebs jobs'
+    regex: '.*_(?i)ebs_.*.*'
+  storage_drivers:
+    description: 'All Storage Driver Jobs'
+    regex: '.*_(?i)storage_(?i)driver_.*.*'
+  AWS:
+    description: 'All AWS Jobs'
+    regex: '.*_(?i)aws_.*.*'
+  Rackspace:
+    description: 'All Rackspace Jobs'
+    regex: '.*_(?i)rackspace_.*.*'
+  CentOS:
+    description: 'All CentOS jobs'
+    regex: '.*_(?i)centos_.*.*'
+  Trusty:
+    description: 'All Ubuntu Trusty jobs'
+    regex: '.*_(?i)trusty_.*.*'
+  Vivid:
+    description: 'All Ubuntu Vivid jobs'
+    regex: '.*_(?i)vivid_.*.*'
+  Wily:
+    description: 'All Ubuntu Wily jobs'
+    regex: '.*_(?i)wily_.*.*'
```

Comment:
that would mean renaming all the jobs, and lose history.
I also think this might be a bit of a bike shed issue, as nothing prevents a future PR from suggesting the opposite